### PR TITLE
Fixes adamantine slime (crystalline) crossbreeds [#10551]

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation

### Intent of your Pull Request
Bugged PR: https://github.com/yogstation13/Yogstation/pull/10551
This PR fixes the cores not appearing, as well as fixing few bugs that i noticed during bugtesting.

### Why is this change good for the game?
Fixes good. Bugs bad.
Technically improves RP.

# Wiki Documentation
https://wiki.yogstation.net/wiki/Guide_to_xenobiology
I didn't change the effects of crystals, so wiki is okay.

### Briefly describe your PR and the impacts of it, in layman's terms. 
Adds a missing icon for crossbreed.
Adds description to slime cores, as well as crystals themselves.
Re-adjusts crystal integrity so it doesn't get destroyed by accident.
Correctly puts human players into stasis when they're being transformed into an animal, so they don't suffocate.
Fixed gold crystals being able to turn you into a missing mob.
Red crystal got more organic organs added, i also fixed it not collecting blood trails.

### What should players be aware of when it comes to the changes your PR is implementing?
That they don't get blueballed when making a crossbreed found on wiki.

### What general grouping does this PR fall under? 
Xenobio changes and bugfixes.

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
No, it's just a bugfix.

# Changelog
:cl:  
bugfix: Fixed crossbreeds not appearing as well as some associated bugs.
tweak: Tweaked values for crystal integrity, red crystal gain from collecting blood, given it more organs to create, buffed adamantine armor aura.
imageadd: Added missing icon for the crossbred core.
spellcheck: Added missing decriptions and names.
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
There you go, i coded it myself.

File: code/modules/research/xenobiology/crossbreeding/crystalized.dm [hunk 1]
```diff
@@ -1,106 +1,138 @@
...
+*/
+
+/obj/item/slimecross/crystalized
+	name = "crystalized extract"
+	desc = "It's crystalline,"
+	effect = "crystalline"
+	icon_state = "crystalline"
+	var/obj/structure/slime_crystal/crystal_type
+
+/obj/item/slimecross/crystalized/Initialize()
```
Comment (Bibby0110) [inline code/modules/research/xenobiology/crossbreeding/crystalized.dm]: unneeded, everything does this by default.
If this doesn't it's been overwritten somewhere else and this would just cause confusion.

File: code/modules/research/xenobiology/crossbreeding/_structures.dm [hunk 2]
```diff
@@ -1,577 +1,634 @@
...
+	name = "slimic pylon"
+	desc = "Glassy, pure, transparent. Powerful artifact that relays the slimecore's influence onto space around it."
+	max_integrity = 30
+	anchored = TRUE
+	density = TRUE
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "slime_pylon"
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	///Description of the crystal's effects.
+	var/effect_desc = "null"
```
Comment (Bibby0110) [inline code/modules/research/xenobiology/crossbreeding/_structures.dm]:
```suggestion
	var/effect_desc

```

File: code/modules/research/xenobiology/crossbreeding/_structures.dm [hunk 3]
```diff
@@ -1,577 +1,634 @@
...
+
+	var/mob/living/carbon/human/human = affected_mob
+	human.dna.species.brutemod *= 0.5  //Hefty resistance, great for point defence.
+	human.dna.species.burnmod *= 0.5
+
+/obj/structure/slime_crystal/adamantine/on_mob_leave(mob/living/affected_mob)
+	if(!ishuman(affected_mob))
+		return
+	var/mob/living/carbon/human/human = affected_mob
+	human.dna.species.brutemod /= 0.5
```
Comment (Bibby0110) [inline code/modules/research/xenobiology/crossbreeding/_structures.dm]:
```suggestion
	human.dna.species.brutemod *= 2

```

File: code/modules/research/xenobiology/crossbreeding/_structures.dm [hunk 4]
```diff
@@ -1,577 +1,634 @@
...
+	human.dna.species.burnmod /= 0.5
```
Comment (Bibby0110) [inline code/modules/research/xenobiology/crossbreeding/_structures.dm]:
```suggestion
	human.dna.species.burnmod *= 2

```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
looks owkay

**Comment by ChesterTheCheesy** [comment]
fuck wrong branch

**Comment by ChesterTheCheesy** [comment]
how the fuck do i revert commits

**Comment by ChesterTheCheesy** [comment]
Still learning how to do github, soz guys.
